pkg/grpc/tracing/stats.go

```diff
@@ -19,6 +19,9 @@ var (
 
 func InitMetrics(registry *prometheus.Registry) {
 	registry.MustRegister(LoaderStats)
+
+	// NOTES:
+	// * Rename process_loader_stats metric (to e.g. process_loader_events_total) and count label (to e.g. event)?
```

Comment:
Does it make sense to link GH issues here?

Reply:
yeah, I think so, just I'm not 100% sure which of these points are worth implementing and which not. Many of these are technically breaking changes, but also many metrics are probably used only by Tetragon developers so :woman_shrugging: I need one more pass over the metrics and maybe another pair of eyes, then I'll open issues for improvements.